packages/build/src/plugins_core/deploy/build_service.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+option go_package = "github.com/netlify/buildbot/buildservice/proto";
+
+package buildservice;
+
+// Service to communicate between Netlify Build and the buildbot
+service BuildService {
+  // Request the buildbot to deploy the site
+  rpc RunPostDeploy(PostDeployRequest) returns (PostDeployResponse) {}
+}
+
+message PostDeployRequest {
+  // Whether the request should wait for the deploy (including post-processing)
+  // to finish before completing
+  bool await_live = 1;
```

Comment:
`await_live` will default to `false`. This seems like the default we would like to have. Correct?

Reply:
Yes.
Please note also that, in practice, the default value will never be used (`await_live` is always explicitly set).